homeassistant/components/ring/__init__.py

```diff
@@ -30,42 +26,9 @@
 
 PLATFORMS = ("binary_sensor", "light", "sensor", "switch", "camera")
 
-CONFIG_SCHEMA = vol.Schema(
-    {
-        vol.Optional(DOMAIN): vol.Schema(
-            {
-                vol.Required(CONF_USERNAME): cv.string,
-                vol.Required(CONF_PASSWORD): cv.string,
-            }
-        )
-    },
-    extra=vol.ALLOW_EXTRA,
-)
-
 
 async def async_setup(hass, config):
     """Set up the Ring component."""
-    if DOMAIN not in config:
-        return True
-
-    def legacy_cleanup():
-        """Clean up old tokens."""
-        old_cache = Path(hass.config.path(".ring_cache.pickle"))
```

Comment:
Shouldn't we keep this around for a little longer?

Reply:
Reinstated.

Reply:
You've also re-instated:

```python
    if DOMAIN not in config:
        return True
```

Considering the change, the re-instated code is now unreachable.

Reply:
Yeah, if the user never had a manual config, they never had that file.

Reply:
But if they removed the manual config, the file is never cleaned.
If that was the goal 👍

Reply:
I think that's ok. It's been cleaning up/not used since 104 and it's not the end of the world if it isn't cleaned up. I just don't like going into a thread and doing I/O unnecessarily for the rest of the integrations life.